AFQ/tests/test_api.py

```diff
@@ -604,6 +604,15 @@ def test_AFQ_pydra():
     pga.export("dti_fa")
 
 
+def test_AFQ_filterb():
+    _, bids_path, _ = get_temp_hardi()
+    myafq = GroupAFQ(
+        bids_path=bids_path,
+        preproc_pipeline='vistasoft',
+        max_bval=1000)
```

Comment:
Would this actually test this functionality? This data doesn't have b-vals above 1000 does it? I'll merge for now but just wondering whether there's a better test.